java/core/src/main/java/module-info.java

```diff
@@ -0,0 +1,10 @@
+// Protocol Buffers - Google's data interchange format
+// Copyright 2008 Google Inc.  All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+open module com.google.protobuf {
+    exports com.google.protobuf;
+}
```

Comment:
Protobuf core module definition; very simple and kept `open` for reflective access, which I understand to be frequent for Protobuf's use case.